package.json

```diff
@@ -19,6 +19,7 @@
 		"docs": "typedoc source/index.ts",
 		"build": "del-cli dist && tsc",
 		"prepublishOnly": "npm run build",
+		"postpublish": "gh-pages --dist docs --no-history --message \"Deploy documentation\"",
```

Comment:
Shouldn't this also run `nom run docs` first?

Reply:
Ideally, it would run `npm run build`, `npm run docs` and then the publish command.